application/lib/tutorial/two/tutorial_two.dart

```diff
@@ -46,13 +47,23 @@ class TutorialTwo extends StatelessWidget {
               child: const Text("Explore Menu Module"),
             ),
             ElevatedButton(
-                onPressed: () => {
-                      Navigator.push(
-                        context,
-                        MaterialPageRoute(builder: (context) => AdjustSlider()),
-                      )
-                    },
-                child: Text("Adjust slide module"))
+              onPressed: () => {
+                Navigator.push(
+                  context,
+                  MaterialPageRoute(builder: (context) => AdjustSlider()),
+                );
```

Comment:
The semicolon causes build error on my end. Does it not cause this error on your end?

Reply:
oops was updating this on my other device that didn't do compile checks properly, fixed

Reply:
though this seems to be coming from the ApplicationPort branch rather than this one, did we merge in a build error previously?